### PR TITLE
Shoppingcartfix

### PR DESCRIPTION
**Pull request by harjeet904**
changes are made please approve

### CODE REVIEW

File: ShoppingCartDemo-Start/Program.cs [hunk 1]
```diff
@@ -30,7 +30,7 @@ static void Main(string[] args)
                 userInput = GetPurchaseOption();
 
                 // TODO1: Uncomment to be able to respond to user input
-                // RespondToMenu(userInput);
+                 RespondToMenu(userInput);
```
Comment (eka-tjung) [inline ShoppingCartDemo-Start/Program.cs:33]: This is what I want

File: .gitignore [hunk 2]
```diff
@@ -1,4 +0,0 @@
-#Ignore shopping cart
-ShoppingCartDemo-Start/.vs
-ShoppingCartDemo-Start/obj
-ShoppingCartDemo-Start/bin
```
Comment (eka-tjung) [inline .gitignore:4]: Shouldn't remove these.
It creates a lot of noise

**Comment by harjeet904** [comment]
Thank you !!

**Comment by harjeet904** [comment]
Code need review again

**Pull request merged**